pull_request_template.md

```diff
@@ -0,0 +1,16 @@
+{package} {version} {:snowflake:}
+
+**Destination channel:** {Snowflake | defaults}
+
+### Links
+
+- [{ticket_number}]() 
+- [Upstream]()
+- [conda-forge]()
+- [pypi]()
+- Dependencies:
+  - ...
+
+### Changes:
```

Comment:
I think it would be useful for people to list why they've made the changes, rather than just list the changes. (The changes can be seen from the PR changes, anyway.) Would it be possible to add a short note saying that?

Reply:
I agree this and the note for the dependencies that exist on staging channels, etc. will be helpful.

Reply:
I'm wary of clutter, but I adjusted the wording of the heading to "Explanation of Changes."

Reply:
sounds good